### DISCUSSION TITLE
Allow changing image aspect in AvatarView

### DISCUSSION THREAD

**Discussion by MahmoudAlEssawi** [CommunityToolkit/Maui#1482] [New Feature Discussions]
Add Aspect property with ImageSource Aspect enumeration to AvatarView

**Comment by GeorgeLeithead** [comment]
@MahmoudAlEssawi I don't see any reason why we couldn't provide the ability to set the `Aspect` of the AvatarView.  It MAY cause issues when used in conjuction with a `border`.  It should be reasonably easy to implement, as currently we default the `aspect` to `AspectFill`.

Before spending time on this, can I ask what the reason for needing `aspect` is specifically?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers.
